### ISSUE TITLE
Anyone have a CSLA assessment quiz I could use

### ISSUE THREAD

**Issue by swegele** [MarimerLLC/cslaforum#916]
Going to be hiring a CSLA developer, or one that is willing to learn CSLA.
Before starting the search, it would be helpful to have a quiz or assessment of some kind to use an one kind of measure of the persons skills (either up front or after appropriate training).  

Anyone have something written up already and willing to share?

Thanks ahead of time!

**Comment by Chicagoan2016** [comment]
@swegele , I have to admire your company, you are willing to provider CSLA training! I wish I could find a company like that when I was starting out.
I don't have something ready, but based on my experience, if the candidate has some exposure to OO programming that would be of tremendous help in learning CSLA. I have also noticed that not every developer is willing to learning OO programming, just my two cents.

**Comment by swegele** [comment]
@Chicagoan2016 Yes, I am quite privileged to work for a company that gives support at the highest levels for good "doctrinaire" (as my CEO says) architecture including CSLA.  Finding developers willing to learn OO and CSLA has been a big challenge!  But, I promised I would keep trying and so here we go again :-)

I suppose I should have mentioned in my post to y'all...if you know of a good developer with CSLA experience (or willing to learn) then send them my way.  Company is headquartered in Mesa AZ, and there might be a chance of tele-commuting for the right candidate after proving themselves.

BTW I just read your postings about CSLA.  I've been using CSLA since VB6 (even crazier...my dad , now retired, used it before me).  Obviously I recommend it.  For LOB apps, if you take the time to learn it and it's patterns, you can become **ridiculously** productive, disciplined, and you nicely insulate your biz logic from the madness in the UI layer that will light your hair on fire.
It seems to me that @rockfordlhotka and our CSLA community managed to nail the sweet spot filling a need in this space because this framework has weathered HUGE industry changes ebbing and flowing.  Now with WebAssembly  <queue electric guitars> I'm back in the saddle again!  I am very excited.

**Comment by Chicagoan2016** [comment]
@swegele , that's the coolest family history of using CSLA, love it.
I have never been to Mesa, AZ but it sure seems like a fun place!
